syncopy/tests/test_tools.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# Test shared tools.
+#
+
+import syncopy as spy
+import copy
+
+class TestTools:
+
+    def test_structdict_shallow_copy(self):
+        """Test for fix of issue #394: 'Copying a spy.StructDict returns a dict'."""
+        cfg = spy.StructDict()
+        cfg.a = 0.5
+        cfg.b = "test"
+        cfg.c = [1, 2, 3]
+        assert type(cfg) == spy.shared.tools.StructDict
+
+        cfg2 = cfg.copy()
+        _ = spy.StructDict()
+
+        assert type(cfg2) == spy.shared.tools.StructDict
+        assert cfg.a == 0.5
+        assert cfg.b == "test"
+        assert cfg.c == [1, 2, 3]
+
+        assert cfg2.a == cfg.a
+        assert cfg2.b == cfg.b
+        assert cfg2.c == cfg.c
+
+        # Check the list was shallow-copied.
+        cfg.c.append(4)
+        assert cfg2.c == cfg.c
+
+    def test_structdict_from_dict(self):
+        my_dict = {'a': 0.5, 'b': 'test', 'c' : [1, 2, 3]}
+        cfg = spy.StructDict(my_dict)
+        assert type(cfg) == spy.shared.tools.StructDict
+        assert cfg.a == 0.5
+        assert cfg.b == "test"
+        assert cfg.c == [1, 2, 3]
+
+    def test_structdict_from_dict_copy(self):
```

Comment:
Is that test really necessary? Once the `cfg` is a `StructDict`, this test just repeats the standard copy test no?!

Reply:
yes, I will remove it. I just had the impression in between that something was fishy, but it had a different reason.

Reply:
its removed